api/runner/async_runner.go

```diff
@@ -52,8 +51,7 @@ func getCfg(task *models.Task) *Config {
 		Timeout: time.Duration(*task.Timeout) * time.Second,
 		ID:      task.ID,
 		AppName: task.AppName,
-		Stdout:  stderr,
-		Stderr:  stderr,
+		Stdout:  ioutil.Discard,
```

Comment:
@ccirello If read the previous impl of `getCfg()` you'll see that we was sending the output to the stderr used to log. What is the benefit of logging the async result?